control-plane/api-gateway/controllers/index.go

```diff
@@ -325,3 +331,15 @@ func filtersForHTTPRoute(o client.Object) []string {
 	}
 	return filters
 }
+
+func gatewayForGatewayPolicy(o client.Object) []string {
+	gatewayPolicy := o.(*v1alpha1.GatewayPolicy)
+
+	targetGateway := gatewayPolicy.Spec.TargetRef
```

Comment:
~~is this the target gateway? the TargetRef should be a listener on a gateway right?~~ nevermind, the target ref will be a gateway with the listener as part of the section name